include/yaclib/util/detail/atomic_counter.hpp

```diff
@@ -33,6 +33,20 @@ struct AtomicCounter : CounterBase {
     }
   }
 
+  void IncRef(std::size_t cnt) noexcept {
+    count.fetch_add(cnt, std::memory_order_relaxed);
+  }
+
+  void DecRef(std::size_t cnt) noexcept {
+    if (SubEqual(cnt)) {
+      Deleter::Delete(*this);
+    }
+  }
```

Comment:
Some suggestion:
1. always inline
2. use it in virtual IncRef/DecRef
3. count instead of cnt